operations/alloy-mixin/alerts/opentelemetry.libsonnet

```diff
@@ -11,11 +11,12 @@ local alert = import './utils/alert.jsonnet';
         alert.newRule(
           'OtelcolReceiverRefusedSpans',
           if enableK8sCluster then
-            'sum by (cluster, namespace, job) (rate(receiver_refused_spans_ratio_total{}[1m])) > 0'
+            'sum without (cluster, namespace, job, instance) (rate(receiver_refused_spans_ratio_total{}[1m])) > 0'
           else
-            'sum by (job) (rate(receiver_refused_spans_ratio_total{}[1m])) > 0'
+            'sum without (job, instance) (rate(receiver_refused_spans_ratio_total{}[1m])) > 0'
```

Comment:
Not sure, I follow this. I think this will aggregate excluding job, and instance. Granted I havent used without often.

Reply:
You are right, this won't achieve what I wanted. Will fall back to use by () and keep the annotations changes only.